packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -442,6 +403,8 @@ const initEnhancedCoverageSchema = z.object({
 /** ---------------------------------------------------------------------------
  * POST /internal/patient/enhance-coverage
  *
+ * @deprecated REMOVE THIS AND RELATED/DOWNSTREAM CODE
```

Comment:
Flagging EC endpoints that should be removed as part of https://github.com/metriport/metriport-internal/issues/1543